modules/reliability/php/NDB_Menu_Filter_reliability.class.inc

```diff
@@ -69,32 +69,41 @@ class NDB_Menu_Filter_reliability extends NDB_Menu_Filter
         foreach($pscRows AS $row) {
             $this->centerIDMap[$row['CenterID']] = $row['Name'];
         }
-        
+
         $this->_populate_reliability_table();
 
-        $query = " FROM candidate, session, flag, reliability WHERE reliability.CommentID=flag.CommentID AND candidate.CandID=session.CandID AND session.ID=flag.SessionID";
+        $query = " FROM session, flag, " .
+            "reliability LEFT JOIN psc p ON reliability.reliability_center_id=p.CenterID, " .
```

Comment:
I think there's a syntax error here (comma at the end of the line in the ON clause.)

As long as you're updating that, can you put brackets around the ON clause for readability? And it might be a good idea to make the query consistently ANSI join syntax ("LEFT JOIN ... ON") instead of half cross joins (comma separating tables w/ where clause) and half ANSI joins.